### PR TITLE
MWPW-156810: adding caas config change for live CTAs

### PR DESCRIPTION
**Pull request by sheridansunier**
As a Virtual Events Product Manager, I want to have a way for a different destination (e.g anchor link) and CTA copy when an event card is live so that when we use event cards to promote the upcoming homepage schedule, a click to the card will take the user to the specific destination (specifically to the marquee video player on the homepage).  

This PR adds the configuration change necessary to facilitate the above in CAAS.

Resolves: [MWPW-145727](https://jira.corp.adobe.com/browse/MWPW-145727)
[MWPW-156810](https://jira.corp.adobe.com/browse/MWPW-156810)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page?martech=off
- After: https://MWPW-145727--milo--sheridansunier.hlx.page?martech=off

### CODE REVIEW

**Review by mokimo — APPROVED** [review]

File: test/blocks/caas-config/expectedConfigs/defaultConfig.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const defaultConfig = {
     reservoir: { sample: 3, pool: 1000 },
     ctaAction: '_self',
     additionalRequestParams: {},
+    altCtaOnLive: false
```
Comment (overmyheadandbody) [inline test/blocks/caas-config/expectedConfigs/defaultConfig.js:43]: Comma needed, as ESLint is also suggesting

**Comment by sanrai** [comment]
@sheridansunier is out on PTO and will be back on 08/20 and add fix then (she doesn't have access to internet on her PTO):

The fix she needs is this:
image

**Comment by mokimo** [comment]
@sanrai you could close this PR and open a new one

**Comment by sheridansunier** [comment]
to clarify it was "reliable access to internet on her pto" - @mokimo the unit tests should be fixed now

**Review by sanrai — APPROVED** [review]

**Review by cmiqueo — APPROVED** [review]

**Pull request merged**